extending.rst

```diff
@@ -11,3 +11,4 @@ This section provides an introduction to extension capabilities of Conan:
    extending/custom_settings
    extending/python_requires
    extending/hooks
+   extending/sanitizers
```

Comment:
I think this should go inside "custom settings"

Reply:
Ok, now I understand why it is not there. Maybe this should not be in extending Conan but in the "how to" section

Reply:
I think the "how to" section would be a good place for this information too.
Maybe we should move it...